mitodl/mitxonline
# Adjust styles of Refine dashboard to be more MIT

#### What are the relevant tickets?
N/A

#### What's this PR do?
This PR updates a few styles so that it uses the MIT logo and colors instead of Refine's.

#### How should this be manually tested?
Run the app and verify the refine dashboard is still functional and looks like the screenshots.

#### Screenshots (if appropriate)
![Screenshot 2022-06-03 at 16-30-37 refine - A React-based framework for building data-intensive applications in no time](https://user-images.githubusercontent.com/28598/171947832-d4a34460-91e0-44e0-9cfa-1823888f259a.png)
![Screenshot 2022-06-03 at 16-28-34 refine - A React-based framework for building data-intensive applications in no time](https://user-images.githubusercontent.com/28598/171947834-fefb85e5-d413-4322-90a6-a146fdf8b316.png)
